src/containers/Triggers/Triggers.js

```diff
@@ -25,16 +25,16 @@ import { FormattedDate, Table } from '@tektoncd/dashboard-components';
 import { Link as CarbonLink } from 'carbon-components-react';
 
 import { ListPageLayout } from '..';
-import { useTriggers } from '../../api';
-import {
-  getSelectedNamespace,
-  isWebSocketConnected as selectIsWebSocketConnected
-} from '../../reducers';
+import { useSelectedNamespace, useTriggers } from '../../api';
+import { isWebSocketConnected as selectIsWebSocketConnected } from '../../reducers';
 
 function Triggers(props) {
-  const { filters, intl, namespace, webSocketConnected } = props;
+  const { filters, intl, match, webSocketConnected } = props;
   useTitleSync({ page: 'Triggers' });
 
+  const { selectedNamespace } = useSelectedNamespace();
+  const namespace = match.params.namespace || selectedNamespace;
```

Comment:
Nit. Think I prefer the style you used here: https://github.com/tektoncd/dashboard/pull/2141/files?diff=split&w=1#diff-d4255f636c1984d552c8f721d92957aa4156f5469cc8fcd47303a72faa5aaf42R36. Same effect

`const { namespace = selectedNamespace } = props.match.params;`

Reply:
Yeah I think I switched to that style halfway through then forgot to go back and update the previous instances 👍